### PR TITLE
:flag_pl: Fixes to polish

### PR DESCRIPTION
**Pull request by KaspianDev**
Fixed some typos, changed some stuff

We got a number one victory royale
Yeah, Fortnite, we 'bout to get down (Get down)
Ten kills on the board right now
Just wiped out Tomato Town
My friend just got downed
I revived him, now we're heading south-bound
Now we're in the Pleasant Park streets
Look at the map, go to the marked sheet

Take me to your Xbox to play Fortnite today
You can take me to Moisty Mire, but not Loot Lake
I really love to chug jug with you
We can be pro Fortnite gamers

### CODE REVIEW

**Review by KaspianDev — COMMENTED** [review]
Very good

**Comment by TBlueF** [comment]
I don't know what this description is, but it gives me the impression you are just trolling.
Please change that and describe in a meaningful way what you have changed and why, else i will have to deny this pull-request..

**Comment by KaspianDev** [comment]
i just had nothing to say, i fixed some typos and stuff thats all

**Review by TBlueF — CHANGES_REQUESTED** [review]
Okay, thanks!

One thing before i merge this: 
i noticed the translation-autor-mention on the info-page.

I explained [here](https://github.com/BlueMap-Minecraft/BlueMapVue/pull/18#issuecomment-815676366) why i would like to avoid this. I also got permission from Artuu there, to to remove that mention again.

So: Can you please do that and remove their mention and your own?
Thanks :)

File: public/lang/pl.js [hunk 1]
```diff
@@ -145,7 +145,7 @@ export default {
 			<td>
 				<kbd>Lewy ALT</kbd> + <kbd>WASD</kbd>,
 				<br />
-				<kbd>Lewy ALT</kbd> + <kbd>strzałki</kbd>,
+				<kbd>Lewy ALT</kbd> + <kbd>Strzałki</kbd>,
```
Comment (xartuu) [inline public/lang/pl.js:148]: In the 132nd line, you can also write `Strzałki` with a capital letter.

**Review by xartuu — CHANGES_REQUESTED** [review]

File: public/lang/pl.js [hunk 2]
```diff
@@ -98,7 +98,7 @@ export default {
         position: "Pozycja",
         chunk: "Chunk",
         region: {
-            region: "Region",
```
Comment (xartuu) [inline public/lang/pl.js]: I think the word `region` is more correct, since the minecraft world contains regions, not territories (*tereny*).

**Review by xartuu — CHANGES_REQUESTED** [review]
Simply put, the word `region` is used by the minecraft world, just like the word `chunk`.

**Comment by xartuu** [comment]
Thank you @kacperleague9 for correcting my minor oversights in my translation.

**Comment by KaspianDev** [comment]
Done

File: public/lang/pl.js (hunk 1 above)
Comment (KaspianDev) [reply public/lang/pl.js:148]: okay

File: public/lang/pl.js [hunk 3]
```diff
@@ -164,8 +164,6 @@ export default {
 </p>
 <br><hr>
 <p>
-Ta mapa została wygenerowana z &#9829; używając <a href="https://bluecolo.red/bluemap">BlueMap</a>. Autor tłumaczenia - <a href="https://artuu.pl">Artuu</a>.
-</p>
-		`
+Ta mapa została wygenerowana z &#9829; używając <a href="https://bluecolo.red/bluemap">BlueMap</a>.</p>
```
Comment (TBlueF) [inline public/lang/pl.js]: I think you accidentally removed the ``` ` ``` as well, which is needed to close the multiline string ^^

**Review by TBlueF — CHANGES_REQUESTED** [review]

**Comment by KaspianDev** [comment]
oops, fixed

**Review by TBlueF — APPROVED** [review]

**Pull request merged**